docs/edgecases.md

```diff
@@ -52,99 +69,118 @@ building
+  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
+  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
+ModuleNotFoundError: No module named 'setuptools'
+
+error (ignored): error: cannot unlink '/tmp/nix-build-python3.10-django-4.1.3.drv-1/Django-4.1.3': Directory not empty
+error: 1 dependencies of derivation '/nix/store/hz4b87s99s1lwiz2m0vwilhlh6rlfj64-python3-3.10.8-env.drv' failed to build
+error: 1 dependencies of derivation '/nix/store/60gaxbdf11lhaj9xg50cf0rr6x5v8v1z-nix-shell-env.drv' failed to build
 ```
+
+As you can see on the fourth last line it's missing `setupools` which in this case is our missing `PACKAGENAME`.
+
 </details>
 
-In order to be able to build `extralib` we should modify our nix definition as follows:
+In order to be able to build `django-floppyforms` we should modify our nix definition as follows:
+
+``` nix
+poetry2nix.mkPoetryApplication {
+  projectDir = ./.;
+  overrides = poetry2nix.defaultPoetryOverrides.extend
+    (self: super: {
+      django-floppyforms = super.django-floppyforms.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.buildtools ];
+        }
+      );
+    });
+}
 ```
+
+This override will instruct the underlying build logic to include the additional build dependency into the inputs of `django-floppyforms`.
+It might help to know that you don't need to use the full package name like `python39Packages.setupools` but can just use `setuptools` directly.
+However you have to use the name like defined in `Nixpkgs` so something like `flit_scm` becomes `flit-scm`.
+
+It can happen that you need multiple overrides for your project, just work through one after the other and your project should build at the end.
+Your file might then look something like this:
+
+``` nix
 poetry2nix.mkPoetryApplication {
   projectDir = ./.;
-  overrides = poetry2nix.defaultPoetryOverrides.extend ( self: super: {
-    extralib = super.extralib.overridePythonAttrs (
-      old: {
-        buildInputs = (old.buildInputs or [ ]) ++ [ self.poetry ];
-      }
-    );
-  });
+  overrides = poetry2nix.defaultPoetryOverrides.extend
+    (self: super: {
+      first-dependency = super.first-dependency.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.buildtools ];
+        }
+      );
+      second-dependency = super.second-dependency.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.pdm ];
+        }
+      );
+    });
 }
 ```
-This override will instruct underlying build logic to include additional build dependency into inputs of `extralib`.
+
+We recommend that you contribute your changes to `poetry2nix` so that other users can profit from them as well.
+The file with the upstream overrides can be found here: https://github.com/nix-community/poetry2nix/blob/master/overrides/build-systems.json
```

Comment:
Maybe add a few words about how to actually add overrides to that?

Reply:
Do you have something specific in mind?

Reply:
Just a simple explanation of the format of the file and where things need to be added.

Reply:
That'd be nice, but I don't think it's needed for accepting this PR. The format is IMO quite self-evident from the existing content of that file.

Reply:
I added a sentence about the format.